Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Goblin Loan Decision

This is the Goblins Loans system,Check for bugs or missing properties,also open to any suggestion


## Changelog:
- remade the whole thing.


## Developer changelog:
- remade it


## Tests:


# How to test:
Play as Anyone (not evil relegion or tribal) and see if you can take a loan,then switch to a tribal/evil relegion character and see if the decision is still there.
